arrays_strings/unique_chars/unique_chars_solution.ipynb

```diff
@@ -278,22 +278,23 @@
   }
  ],
  "metadata": {
+  "celltoolbar": "Edit Metadata",
```

Comment:
It's ok, can you add a commit reverting this change to the PR?  I will then squash it to make it a single, clean commit.